lib/util/filterCollection.js

```diff
@@ -1,23 +1,19 @@
-const { v4: uuidv4 } = require('uuid');
+const filterCollection = (collectionDefinition, suites) => {
+    const { item: allItems, ...everythingElse } = collectionDefinition;
+    const filtredItems = [];
 
-module.exports = {
-    filterCollection: (collectionDefinition, suites) => {
-        const { item: allItems, ...everythingElse } = collectionDefinition;
-        const filtredItems = [];
+    // TODO: Add new step in json suite - reflect this behaivor in test
+    suites.forEach(value => {
+        const itemCollection = { ...allItems.find(item => item.name in value) };
+        filtredItems.push(itemCollection);
+    });
 
-        //TODO: Add new step in json suite - reflect this behaivor in test
-        suites.forEach((value) => {
-            const itemCollection = Object.assign(
-                {},
-                allItems.find((item) => item.name in value)
-            );
-            filtredItems.push(itemCollection);
-        });
-
-        if (Array.isArray(filtredItems) && filtredItems.length == 0) {
-            throw new Error('');
-        }
-
-        return Object.assign(everythingElse, { item: filtredItems });
+    if (Array.isArray(filtredItems) && filtredItems.length === 0) {
+        // TODO: Fill error
+        throw new Error('');
```

Comment:
O que deveria entrar nessa mensagem?

Reply:
My fault, this exception shall rise this message "Suit file, not provide a valid requests". I fix this now.